docs/configuration.rst

```diff
@@ -971,3 +971,100 @@ This must be used in conjunction with ``encrypted = true`` and must have a custo
 See `Encrypted EBS with a Custom KMS Key <_encrypted_ebs>`. ::
 
     ebs_kms_key_id = xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+
+
+.. _fsx_section:
+
+FSx
+^^^
+Configuration for an attached FSx Lustre file system.
+
+Create a new file system, by omitting ``fsx_fs_id``, or use an existing one by specifying ``fsx_fs_id``. ::
+
+    [fsx fs]
```

Comment:
Shall we have 2 separate examples: one with fs_id and one without to better highlight what are the parameters that are actually used in each case? Cause I expect users to simply copy paste this section and ignoring the rest :)

Reply:
good idea